### PR TITLE
Bugfix for ampersand escaping

### LINKED ISSUE #80: Undesirable ampersand escaping during processing
A string like "chips & tacos" ends up as "chips \u0026 tacos".

The culprit is Go's [`json.Marshal`](https://github.com/golang/go/blob/master/src/encoding/json/encode.go#L46-L53).

### PR DESCRIPTION
**Pull request by jsccast**
As you can imagine from the changes here, I'm a little fearful.

### CODE REVIEW

**Comment by jsccast** [comment]
See #80.

File: dsl/bindings.go [hunk 1]
```diff
@@ -123,14 +124,20 @@ func (bs *Bindings) SerialSub(ctx *Ctx, serialization string, payload interface{
 		s = str
 	} else {
 		structured = true
-		js, err := json.Marshal(&payload)
+		log.Printf("DEBUG 1 %#v", payload)
```
[the remark this answers is not in the document]
Comment (robertfarnum) [reply dsl/bindings.go:127]: @jsccast Above?

[the remark this answers is not in the document]
Comment (jsccast) [reply dsl/bindings.go:127]: Oops. Will fix! Thanks.

**Review by robertfarnum — APPROVED** [review]
Be afraid, very afraid! :-)

**Pull request merged**